src/Microsoft.OData.Client/BatchSaveResult.cs

```diff
@@ -279,6 +283,15 @@ private static string CreateMultiPartMimeContentType()
             return string.Format(CultureInfo.InvariantCulture, "{0}; {1}={2}_{3}", XmlConstants.MimeMultiPartMixed, XmlConstants.HttpMultipartBoundary, XmlConstants.HttpMultipartBoundaryBatch, Guid.NewGuid());
         }
 
+        /// <summary>
+        /// Creates the type of the application/json content.
+        /// </summary>
+        /// <returns>An application/json header</returns>
+        private static string CreateApplicationJsonContentType()

```

Comment:
@KenitoInc Any reason you can't use a `const string` to achieve the same purpose? Use of `string.Format` just introduces a performance penalty IMO.

Reply:
#Resolved